catalog/USFS/USFS_GTAC_TreeMap_v2016.jsonnet

```diff
@@ -0,0 +1,2351 @@
+
+    * The [TreeMap 2016 Data Explorer](https://apps.fs.usda.gov/lcms-viewer/treemap.html) is a web-based application that
+      provides users the ability to view TreeMap attribute data.
+
+    * The [TreeMap Research Data Archive](https://www.fs.usda.gov/rds/archive/Catalog/RDS-2021-0074) for the full dataset download, metadata,
+      and support documents.
+
+    * [TreeMap Raster Data Gateway](https://data.fs.usda.gov/geodata/rastergateway/treemap/) for separated attribute data downloads, metadata, and support documents.
+
+    * [FIA Database Manual version 8](https://www.fia.fs.usda.gov/library/database-documentation/current/ver80/FIADB%20User%20Guide%20P2_8-0.pdf)
+      for more detailed information on the attributes included in TreeMap 2016.
+
+    Contact [sm.fs.gtactreemap@usda.gov](mailto:sm.fs.gtactreemap@usda.gov) with any
+    questions or specific data requests.
+
+    * **Forest Inventory Analysis. 2022a.**
+    Forest Inventory Analysis DataMart.
+    Forest Inventory Analysis DataMart FIADB_1.9.0. 2022. https://apps.fs.usda.gov/fia/datamart/datamart.html.
+
+    * **Ohmann, Janet L and Matthew J Gregory. 2002.**
+    Predictive Mapping of Forest Composition and Structure with Direct Gradient Analysis and Nearest- Neighbor Imputation in Coastal Oregon, USA. 
+    Can. J. For. Res. 32:725-741. [doi: 10.1139/X02-011](https://doi.org/10.1139/X02-011).
+
+    * **Pierce, Kenneth B Jr, Janet L Ohmann, Michael C Wimberly, Matthew J Gregory, and Jeremy S Fried. 2009.**
+    Mapping Wildland Fuels and Forest Structure for Land Management: A Comparison of Nearest Neighbor Imputation and Other Methods.
+    Can. J. For. Res. 39: 1901-1916. [doi:10.1139/X09-102](https://doi.org/10.1139/X09-102).
+
+    * **Riley, Karin L.; Grenfell, Isaac C.; Finney, Mark A.; Shaw, John D. 2021.** 
+    TreeMap 2016: A tree-level model of the forests of the conterminous United States circa 2016. 
+    Fort Collins, CO: Forest Service Research Data Archive. [https://doi.org/10.2737/RDS-2021-0074]https://doi.org/10.2737/RDS-2021-0074.
+
+    * **Wilson, B Tyler, Andrew J Lister, and Rachel I Riemann. 2012.**
+    A Nearest-Neighbor Imputation Approach to Mapping Tree Species over Large Areas Using Forest Inventory Plots and Moderate Resolution Raster Data.
+    Forest Ecol. Manag. 271:182-198. [doi: 10.1016/j. foreco.2012.02.002](https://doi.org/10.1016/j.foreco.2012.02.002).
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(
+        'https://data.fs.usda.gov/geodata/rastergateway/treemap/'),
+    {
+      rel: ee_const.rel.source,
+      href: 'https://data.fs.usda.gov/geodata/rastergateway/treemap/',
+    },
+    ee.link.latest(latest_id, catalog_subdir_url + latest_basename + '.json'),
+    ee.link.predecessor(
+        predecessor_id, catalog_subdir_url + predecessor_basename + '.json'),
+  ],
+  keywords: [
+    'biota',
```

Comment:
note we prefer to be conservative with keywords, that is, not introduce too many. Try using mostly the existing keywords. The checks will complain about the keywords once they get to that stage.